src/python/pants/goal/run_tracker.py

```diff
@@ -85,6 +86,8 @@ def register_options(cls, register):
              help='Number of threads for background work.')
     register('--stats-local-json-file', advanced=True, default=None,
              help='Write stats to this local json file on run completion.')
+    register('--stats-option-scopes-to-record', advanced=True, type=list, default=[],
+             help='Options or option scopes to record in stats on run completion.')
```

Comment:
The format of how to specify an option inside a scope should be documented in this help string.

Reply:
Done